src/transformers/expect-js.js

```diff
@@ -0,0 +1,220 @@
+import { getRequireOrImportName, removeRequireAndImport } from '../utils/imports';
+import finale from '../utils/finale';
+
+/*
+   This a codeshift for 'expect.js' to Jest's expect
+    https://github.com/Automattic/expect.js
+
+    No support yet for the follow expect.js conventions:
+
+    expect({ a: 'b', c: 'd' }).to.only.have.keys(['a', 'c']);
```

Comment:
Please log a warning if this is used. See some of the other transformers where we also log an error if you are using unsupported features. And add a test. : )

And if you want to implement it, the `chai-should` does something similar.